core/domain/suggestion_services.py

```diff
@@ -2388,7 +2388,7 @@ def _update_translation_contribution_stats_models(
             stat.rejected_translations_count)
         stats_model.rejected_translation_word_count = (
             stat.rejected_translation_word_count)
-        stats_model.contribution_dates = stat.contribution_dates
+        stats_model.contribution_dates = sorted(stat.contribution_dates)
```

Comment:
@seanlip this is the only place where we update the `TranslationContributionStatsModels`.  When creating these models for the first time, we don't need to sort contributin_dates, cuz there will be only one element.

Reply:
I'm not sure we should rely on that. Let's add a sorted() there just to be safe as well.

But this looks good too, thanks for adding it.